synapse/storage/databases/main/metrics.py

```diff
@@ -282,8 +282,8 @@ def _generate_user_daily_visits(txn):
             now = self._clock.time_msec()
 
             sql = """
-                INSERT INTO user_daily_visits (user_id, device_id, timestamp)
-                    SELECT u.user_id, u.device_id, ?
+                INSERT INTO user_daily_visits (user_id, device_id, timestamp, user_agent)
+                    SELECT u.user_id, u.device_id, ?, MAX(u.user_agent)
```

Comment:
If you could just add a comment about the SQL to say why its fine to take the MAX that would be great